_posts/2019-04-24-gis-wisdom.md

```diff
@@ -0,0 +1,111 @@
+
+### 3. Extract, Translate, and Load (ETL) 
+{: .text-left}
+
+`being adaptable`
+
+You have probably heard the phrase, 'build once, use many'.  When building data, we say this when we want the output to be adaptable and used for many purposes.  Take road centerlines for example.  This dataset should be designed with many uses in mind (such as routing, mapping, locating, assessment management, etc.).  With a simple ‘extract, translate, and load’ (ETL), the data should meet the needs of many users.  
+
+We should learn to be adaptable as well.  Life is not stagnant, and the ability to change and adapt is beneficial.  Having a single-use in mind has the potential of putting us back at the drawing board, building again.
+
+Learn more about [creating your own spatial ETL tools](http://desktop.arcgis.com/en/arcmap/10.6/extensions/data-interoperability/spatial-etl-tools.htm).
+
+### 4. Open Data
+{: .text-left}
+
+`sharing`
+
+Ideally, as GIS folks, we're all supporting or contributing to open data in some way.  In other words, advancing freely available data.  This philosophy promotes transparency,  innovation, and efficiency.  Data gets better the more it's used, and open data gets used.
+
+On the human side, the more we share, the more we learn.  And when we share, we not only advance the overall knowledge base, we also advance our own. Ideas grow when they are open and shared.  
+
+Explore some of the [existing GIS open data](http://hub.arcgis.com/pages/open-data) or consider contributing to the [Open Street Maps project](https://www.openstreetmap.org/about).
+
+### 5. Editing
+{: .text-left}
+
+`refinement`
+
+The best datasets have lots of edits and editors.  They are continually being reviewed and refined. Think of multi-editing environments or Open Street Maps.  These datasets are so vast and valuable because they are 'live’ and diverse.  Everyone is bringing something unique to the product, which is difficult to produce with a small group of editors.
+
+But it's the refinement process that makes it valuable.  We too can continually refine through learning and sharing and practice and performance.  In turn, this advances our skills and adds to our value.
+
+Brush-up on some [essential editing vocabulary](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/editing/essential-editing-vocabulary.htm).
+
+### 6. Domains
+{: .text-left}
+
+`consistency`
+
+Domains are set up in the database to establish known consistently. They provide quality assurance on the front-end while ensuring consistency on the back-end.  What makes domains so great is that the end-user can reliably expect certain values or results.  Developers and cartographers benefit from this as well. 
```

Comment:
`Domains are set up in the database to establish known consistently.` Is `consistently` the word you want here?